src/ServiceControlInstaller.PowerShell/Cmdlets/AuditInstances/PsAuditInstance.cs

```diff
@@ -53,6 +55,7 @@ public static PsAuditInstance FromInstance(ServiceControlAuditInstance instance)
                 AuditRetentionPeriod = instance.AuditRetentionPeriod,
                 ServiceControlQueueAddress = instance.ServiceControlQueueAddress,
                 EnableFullTextSearchOnBodies = instance.EnableFullTextSearchOnBodies,
+                PersistencePackageName = instance.PersistenceManifest?.Name
```

Comment:
Should we use the display name instead?

Reply:
That's not what the transport one is doing. This might get used for programatic purposes. i.e. Get the persistence of an instance and create a new instance with the same persistence.

Reply:
Fair enough

Reply:
I stand corrected, the transport package name is a display name. I will change this to match.